src/factoids.coffee

```diff
@@ -43,22 +43,22 @@ module.exports = (robot) ->
       to ?= msg.message.user.name
       msg.send "#{to.trim()}: #{fact.value}"
 
-  robot.respond new RegExp("[#{prefix}]([\\w\\s-]{2,}\\w)", 'i'), (msg) =>
+  robot.respond new RegExp("[#{prefix}]([\\w\\s-]{1,}\\w)", 'i'), (msg) =>
```

Comment:
Yes, i have with rubular.com. Maybe you're missing that last `\w` after the curly bracket, which must be matched.

```
# will match
!aaa
!a a
!  a

# won't match
!aa
```